README.md

```diff
@@ -2,40 +2,55 @@
 Let's make a planetary system!
 
 ## Learning Goals
-### Wave 1
-- Create a custom **class**
-- Create and use an **initialize** method in your class
-- Create and use **instance variables** in your class
 
-### Wave 2
-- Use an **array** to store a list of objects
-- Create and use a **method**
+- Practice creating Custom classes
+- Use a hash to initialize an object
+- Create an object which contains a collection of other objects
+- Practice creating instance methods
+
 
 ## Baseline
-- Create a `Planet` class with a name attribute.
-- You should be able to instantiate a new `Planet` object with an associated name.
+- Create a `SolarSystem` class with an `@planets` instance variable.
+- The initialize method should take a collection of planet names and store them in `@planets`
+- Create a method to add a planet to the list.
+- Create a method which will return a list of the planets in this style:
```

Comment:
Can we emphasize that we want the list to be returned (rather than printed), in the same way that we're doing for the Wave 2 requirement?